extensions/amp-auto-ads/0.1/ad-tracker.js

```diff
@@ -161,6 +161,13 @@ export class AdTracker {
  * @return {!Array<!Element>}
  */
 export function getExistingAds(win) {
-  return [].slice.call(win.document.getElementsByTagName('AMP-AD')).concat(
-      [].slice.call(win.document.getElementsByTagName('AMP-A4A')));
+  return [].slice.call(win.document.getElementsByTagName('AMP-AD'))
+      .filter(ad => {
+        // Filters out AMP-STICKY-AD.
+        if (ad.parentElement && ad.parentElement.tagName == 'AMP-STICKY-AD') {
+          return false;
+        }
+        return true;
+      })
+      .concat([].slice.call(win.document.getElementsByTagName('AMP-A4A')));
```

Comment:
That's an issue from long ago. AMP-A4A is implemented, but all with element name `<am-ad>`